tests/behaviour/contracts/external_input_checks/bool_.sol

```diff
@@ -0,0 +1,22 @@
+pragma solidity ^0.8.6;
+
```

Comment:
No need for underscores at the end of contract names because test filter can include directories